$MirOS: ports/print/cups/patches/patch-cups_ipp-support_c,v 1.2 2006/06/21 19:25:34 bsiegert Exp $
--- cups/ipp-support.c.orig	Sat Jul 12 00:48:26 2008
+++ cups/ipp-support.c	Tue Oct 21 22:56:23 2008
@@ -182,7 +182,7 @@ ippErrorString(ipp_status_t error)	/* I 
   * No, build an "unknown-xxxx" error string...
   */
 
-  sprintf(cg->ipp_unknown, "unknown-%04x", error);
+  snprintf(cg->ipp_unknown, sizeof(cg->ipp_unknown), "unknown-%04x", error);
 
   return (cg->ipp_unknown);
 }
